user-manual.md

```diff
@@ -300,338 +243,161 @@ Chatbot: Nimmst du Drogen?
-
-In diesem Beispiel sucht ein non-binärer Jugendlicher eine Notunterkunft. Er nimmt keine Drogen und hat kein Haustier. Das Geschlecht spielt keine Rolle für die Auswahl der Unterkünfte, da es in Nürnberg nur danach geht, ob der Mensch jugendlich ist oder nicht. Der Endpunkt ist eine Auflistung von Notunterkünften die nur Jugendliche ohne Haustier und ohne Drogenprobleme aufnehmen.
-
-#### Beispiel 7 "Notunterkunft mit Tieren":
-
-```
-Eingabe: Ich suche eine Unterkunft.
-Chatbot: Ich benötige noch einige allgemeine Informationen von dir. Diese werden nach dieser Unterhaltung nicht gespeichert.
-Chatbot: Wie alt bist du?
-Eingabe: 32.
-Chatbot: Welchem Geschlecht fühlst du dich zugehörig?
-Eingabe: Männer.
-Chatbot: Hast du ein Haustier?
-Eingabe: Ja.
-Chatbot: Nimmst du Drogen?
-Eingabe: Nein.
-Chatbot: Folgende Informationen wurden erfasst (werden nicht abgespeichert):
-        - Alter: 32
-        - Geschlecht: nännlich
-        - Haustier: ja
-        - Drogen: nein
         - Volljährig: ja
         - Unter 22 (jugendlich): nein
-Chatbot: Brauchst du eine Notunterkunft?
-Eingabe: ja.
-Chatbot: Hier kannst du eine Notunterkunft (Deep Link) für Menschen mit Haustier finden.
+Chatbot: Hier kannst du Notunterkünfte (Deep Link) für Männer finden.
 ```
 
-##### Zusammenfassung Beispiel 7:
-
-In diesem Beispiel sucht ein erwachsener Mann eine Notunterkunft. Er nimmt keine Drogen, hat jedoch ein Haustier. Der Endpunkt ist eine Auflistung von Notunterkünften die Menschen mit Haustier und ohne Drogenprobleme aufnehmen. Hat ein Hilfesuchender ein Haustier und ein Drogenproblem wird bei der Auswahl der Notunterkünfte das Haustier priorisiert. Auch das Alter spielt keine Rolle.
-
-#### Beispiel 8 "Notunterkunft mit Drogenabhängigkeit":
+#### Beispiel 3 "Anfrage nach einer Unterkunft mit direkter Eingabe eines Teils der benötigten Informationen (männlich, volljährig)"
 
 ```
-Eingabe: Ich suche eine Unterkunft.
-Chatbot: Ich benötige noch einige allgemeine Informationen von dir. Diese werden nach dieser Unterhaltung nicht gespeichert.
-Chatbot: Wie alt bist du?
-Eingabe: 32.
-Chatbot: Welchem Geschlecht fühlst du dich zugehörig?
-Eingabe: Männer.
-Chatbot: Hast du ein Haustier?
-Eingabe: Nein.
+Eingabe: Ich bin männlich, 32 und suche einen Platz zum schlafen
+Chatbot: Ok, zum Thema 'Unterkunft' benötige ich noch einige allgemeine Informationen von dir. Diese werden nach dieser Unterhaltung nicht gespeichert.
+Hast du ein Haustier?
+Eingabe: Nein
 Chatbot: Nimmst du Drogen?
-Eingabe: Ja.
+Eingabe: Nein
 Chatbot: Folgende Informationen wurden erfasst (werden nicht abgespeichert):
         - Alter: 32
-        - Geschlecht: nännlich
-        - Haustier: nein
-        - Drogen: ja
-        - Volljährig: ja
-        - Unter 22 (jugendlich): nein
-Chatbot: Brauchst du eine Notunterkunft?
-Eingabe: ja.
-Chatbot: Hier kannst du eine Notunterkunft (Deep Link) für Menschen mit Drogenproblemen finden.
+        - Geschlecht: weiblich
```

Comment:
In diesem Beispiel wird in der Überschrift ein volljähriger Mann beschrieben. Im Chat aber eine Frau aufgelistet. Passt das so?